### PR TITLE
Creation of monster games SS- 350

### PR DESCRIPTION
**Pull request by Manukumar1995**

### CODE REVIEW

File: lib/activity_board.dart [hunk 1]
```diff
@@ -114,20 +125,20 @@ class ActivityBoardState extends State<ActivityBoard> {
                 orientation == Orientation.portrait ? size.width : size.height,
             width:
                 orientation == Orientation.portrait ? size.width : size.height,
-            color: Colors.white,
-            child: AspectRatio(
-                aspectRatio: 1.0,
-                child: Paper(
-                  previewContainerKey: _previewContainerKey,
-                )),
+            child: Paper(
```
Comment (talapadi) [inline lib/activity_board.dart]: Why has the AspectRatio changed?

File: lib/popup_grid_view.dart [hunk 2]
```diff
@@ -71,7 +71,7 @@ class PopupGridViewState extends State<PopupGridView> {
   void initState() {
     super.initState();
     ActivityModel model = ActivityModel.of(context);
-    model.isDrawing = true;
+    model.isDrawing = false;
```
Comment (talapadi) [inline lib/popup_grid_view.dart]: why has this become false

File: lib/popup_grid_view.dart [hunk 3]
```diff
@@ -307,71 +296,143 @@ class PopupGridViewState extends State<PopupGridView> {
...
-                          height: (size.height - size.width) *.13,
-                          width: size.width ,
-                          child: ColorPicker(
-                              orientation: Orientation.portrait, model: model),
-                        ),
-                      ],
-                    )
-                  : Container(),
-            ),
+                color: Color(0xff283744),
```
Comment (talapadi) [inline lib/popup_grid_view.dart]: Have you tested if the old UI works properly?

File: lib/select_sticker.dart [hunk 4]
```diff
@@ -173,7 +269,7 @@ class SelectStickerState extends State<SelectSticker> {
                   default:
                     if (text.startsWith('assets/stickers') ||
                         text.startsWith('assets/svgimage')) {
-                      // model.addSticker(text);
+                      model.addSticker(text, Colors.white, BlendMode.dst);
```
Comment (talapadi) [inline lib/select_sticker.dart]: why has this been uncommented?

File: lib/popup_grid_view.dart (hunk 3 above)
Comment (Manukumar1995) [reply lib/popup_grid_view.dart]: yes old UI working properly

**Pull request merged**